### DISCUSSION TITLE
How to convert string to numeric

### DISCUSSION THREAD

**Discussion by oaklandgit** [andmarti1424/sc-im#696] [Q&A]
Hi,

I imported a csv with a column of dollar values in string format, e.g. `$19.99`.

How can I convert this to its numeric value in order to perform a sum of the row? I read through the help docs and can't find anything. I'm looking for something like `=@tonum(A1)`

Thanks for any help.

**Comment by nkh** [comment]
Hi, I'm conpletely new to sc-im (you've been warned ;) )

What about converting your csv file do the column doesn't have the prepended $ prior to importing it?
